Packages/com.unity.inputsystem/Documentation~/ActionsEditor.md

```diff
@@ -1,29 +1,27 @@
 ---
 uid: input-system-configuring-input
 ---
-# Configuring Input
+# Configuring Input with the Actions Editor
 
-The **Input Actions Editor** allows you to configure [Input Actions](Actions.md) and their associated [Bindings](ActionBindings.md) and [Control Schemes](ActionBindings.md#control-schemes).
+The **Input Actions Editor** allows you to edit [Action Assets](ActionAssets.md), which contain a saved configuration of [Input Actions](Actions.md) and their associated [Bindings](ActionBindings.md), [Action Maps](ActionsEditor.html#configure-action-maps) and [Control Schemes](ActionBindings.md#control-schemes).
```

Comment:
I'd be tempted to split action maps and control schemes onto separate lines as I think there is an opportunity to introduce the differences a little here:

Calling out the actions maps provide a way to group the actions for ease of enabling/disabling groups of actions. 
Control schemes provide a way to enable/disable a set of devices which cut across actions (e.g. enable just gamepad input across move, jump, crouch etc).

Reply:
fixed in [ad3396b](https://github.com/Unity-Technologies/InputSystem/pull/1867/commits/ad3396ba082647ee3cbfc46b84e96edacf424426)